### PR TITLE
server: update balance score to [0,100].

### PR DESCRIPTION
**Pull request by qiuyesuifeng**

### CODE REVIEW

File: server/balancer.go [hunk 1]
```diff
@@ -52,13 +52,13 @@ func newCapacityBalancer(minRatio float64, maxRatio float64) *capacityBalancer {
...
 // calculate the score, higher score region will be selected as balance from store,
-// and lower score region will be balance to store.
+// and lower score region will be balance to store. The score range is (0,100).
 // TODO: we should adjust the weight of used ratio and leader score in futher,
 // now it is a little naive.
 func (cb *capacityBalancer) score(store *storeInfo, regionCount int) float64 {
 	usedRatioScore := store.usedRatio()
 	leaderScore := store.leaderScore(regionCount)
-	score := usedRatioScore*0.6 + leaderScore*0.4
+	score := (usedRatioScore*0.6 + leaderScore*0.4) * 100
```
Comment (siddontang) [inline server/balancer.go]: I think we should use int for (0, 100), not float64 now. 
Btw, (0, 100) means we can't get 0 or 100?

Comment (qiuyesuifeng) [inline server/balancer.go]: Actually we may get 0 score, but it is invalid i think.
float64 can be int in our algorithm.

File: server/balancer.go [hunk 2]
```diff
@@ -55,10 +55,10 @@ func newCapacityBalancer(minRatio float64, maxRatio float64) *capacityBalancer {
 // and lower score region will be balance to store. The score range is (0,100).
 // TODO: we should adjust the weight of used ratio and leader score in futher,
 // now it is a little naive.
-func (cb *capacityBalancer) score(store *storeInfo, regionCount int) float64 {
+func (cb *capacityBalancer) score(store *storeInfo, regionCount int) int {
 	usedRatioScore := store.usedRatio()
 	leaderScore := store.leaderScore(regionCount)
-	score := (usedRatioScore*0.6 + leaderScore*0.4) * 100
+	score := int(usedRatioScore*0.6+leaderScore*0.4) * 100
```
Comment (siddontang) [inline server/balancer.go]: you may always get 0 here.

Comment (qiuyesuifeng) [inline server/balancer.go]: So the comment [0,100] is much better?

File: server/balancer.go [hunk 3]
```diff
@@ -52,13 +52,13 @@ func newCapacityBalancer(minRatio float64, maxRatio float64) *capacityBalancer {
...
-// and lower score region will be balance to store.
+// and lower score region will be balance to store. The score range is (0,100).
 // TODO: we should adjust the weight of used ratio and leader score in futher,
 // now it is a little naive.
-func (cb *capacityBalancer) score(store *storeInfo, regionCount int) float64 {
+func (cb *capacityBalancer) score(store *storeInfo, regionCount int) int {
 	usedRatioScore := store.usedRatio()
 	leaderScore := store.leaderScore(regionCount)
-	score := usedRatioScore*0.6 + leaderScore*0.4
+	score := int(usedRatioScore*0.6+leaderScore*0.4) * 100
```
Comment (siddontang) [inline server/balancer.go]: I think you should unify the score for int, so usedRatio returns int, and leaderScore also returns int.

**Comment by siddontang** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
@shenli  PTAL

File: server/cache.go [hunk 4]
```diff
@@ -482,13 +482,18 @@ func (s *storeInfo) usedRatio() float64 {
...
+}
+
+// leaderScore is the leader peer count score of store, the score range is [0,100].
+func (s *storeInfo) leaderScore(regionCount int) int {
 	if regionCount == 0 {
 		return 0
 	}
 
-	return float64(s.stats.leaderRegionCount) / float64(regionCount)
+	return s.stats.leaderRegionCount / regionCount
```
Comment (shenli) [inline server/cache.go]: Multiple 100.

Comment (qiuyesuifeng) [inline server/cache.go]: Yep, missed it.

**Comment by shenli** [comment]
LGTM

**Pull request merged**